flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/checkpoints/CheckpointStatistics.java

```diff
@@ -229,8 +242,8 @@ public static CheckpointStatistics generateCheckpointStatistics(AbstractCheckpoi
 				checkpointStatisticsPerTask.put(
 					taskStateStat.getJobVertexId(),
 					new TaskCheckpointStatistics(
-							checkpointStats.getCheckpointId(),
-							checkpointStats.getStatus(),
+						checkpointStats.getCheckpointId(),
+						checkpointStats.getStatus(),
```

Comment:
this change also looks unrelated.
it looks like a minor code style fix which we usually do not do to preserve git history.

if we do the code style fixes then it is usually either
- massively in files which have a lot of code style violation in separate commit
- or close to related changes

Reply:
Thanks for pointing out. let me fix this.

Reply:
reverted back.